### PR TITLE
Fix typo in zh-what-is-kubernetes.md

### PR DESCRIPTION
**Pull request by YuzeZhang**

### CODE REVIEW

**Comment by YuzeZhang** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please follow instructions at [git.k8s.io/community/CLA.md#the-contributor-license-agreement](https://git.k8s.io/community/CLA.md#the-contributor-license-agreement) to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.
> 
> * If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * If you signed the CLA as a corporation, please sign in with your organization's credentials at [identity.linuxfoundation.org/projects/cncf](https://identity.linuxfoundation.org/projects/cncf) to be authorized.
> * If you have done the above and are still having issues with the CLA being reported as unsigned, please log a ticket with the Linux Foundation Helpdesk: [support.linuxfoundation.org](https://support.linuxfoundation.org/)
> * Should you encounter any issues with the Linux Foundation Helpdesk, send a message to the backup e-mail support address at: [login-issues@jira.linuxfoundation.org](mailto:login-issues@jira.linuxfoundation.org)

I've signed a CLA

File: content/zh/docs/concepts/overview/what-is-kubernetes.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ Containers are becoming popular because they have many benefits. Some of the con
   容器镜像构建和部署。
 * 关注开发与运维的分离：在构建/发布时而不是在部署时创建应用程序容器镜像，
   从而将应用程序与基础架构分离。
-* 可观察性不仅可以显示操作系统级别的信息和指标，还可以显示应用程序的运行状况和其他指标信号。
+* 可观察性：不仅可以显示操作系统级别的信息和指标，还可以显示应用程序的运行状况和其他指标信号。
```
Comment (mengjiao-liu) [inline content/zh/docs/concepts/overview/what-is-kubernetes.md:123]: This is consistent with the above English version, maybe you can also submit a PR in the English version?

It seems that the English version is also incorrectly formatted.

**Comment by mengjiao-liu** [comment]
Your CLA signature verification failed. You need to check it again. @YuzeZhang

**Review by Arhell — COMMENTED** [review]
/check-cla

**Comment by YuzeZhang** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please follow instructions at [git.k8s.io/community/CLA.md#the-contributor-license-agreement](https://git.k8s.io/community/CLA.md#the-contributor-license-agreement) to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.
> 
> * If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * If you signed the CLA as a corporation, please sign in with your organization's credentials at [identity.linuxfoundation.org/projects/cncf](https://identity.linuxfoundation.org/projects/cncf) to be authorized.
> * If you have done the above and are still having issues with the CLA being reported as unsigned, please log a ticket with the Linux Foundation Helpdesk: [support.linuxfoundation.org](https://support.linuxfoundation.org/)
> * Should you encounter any issues with the Linux Foundation Helpdesk, send a message to the backup e-mail support address at: [login-issues@jira.linuxfoundation.org](mailto:login-issues@jira.linuxfoundation.org)

check

**Pull request closed without merging**